src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -78,31 +139,40 @@
       name: 'approver',
       section: 'requestDetails',
       documentation: `The user that is requested for approval. When set, "group" property is ignored.`,
-      tableCellFormatter: function(approver) {
-        let self = this;
-        this.__subSubContext__.userDAO.find(approver).then((user)=> {
-          if ( user ) {
-            if ( self.__subSubContext__.user.id == user.id ) {
-              self.add(user.legalName);
-            } else {
-              self.add(user.group);
-            }
-          } else {
-            self.add(approver);
-          }
-        });
+      view: function(_, X) {
+        if ( X.data.status === foam.nanos.approval.ApprovalStatus.REQUESTED ) {
+          return {
+            class: 'foam.u2.view.ValueView',
+            data$: X.data$.map((data) => data.REQUESTED)
+          };
+        } else {
+          return { class: 'foam.u2.view.ReferencePropertyView' };
+        }
       },
-      visibility: function(approver) {
-        return approver ?
-          foam.u2.DisplayMode.RO :
-          foam.u2.DisplayMode.HIDDEN;
+      tableCellFormatter: function(approver, data) {
+        let self = this;
+        // If request is REQUESTED, show as Pending
+        // Otherwise, show approver's name
+        if ( data.status === foam.nanos.approval.ApprovalStatus.REQUESTED ) {
```

Comment:
previous logic showed the approvers name if they were logged in. 
Example - as a user that can others approvals - payment-ops (or admin) one cannot tell which approval is for them.

Reply:
Yeah we can revert back to the previous logic I think this was the liquid specific code that had to be added for the demo

Reply:
Feel free to remove